js/common/util.go

```diff
@@ -65,21 +65,15 @@ func ToString(data interface{}) (string, error) {
 }
 
 // RunWithPanicCatching catches panic and converts into an InterruptError error that should abort a script
-func RunWithPanicCatching(logger logrus.FieldLogger, rt *goja.Runtime, fn func() error) (err error) {
+func RunWithPanicCatching(logger logrus.FieldLogger, _ *goja.Runtime, fn func() error) (err error) {
```

Comment:
I specifically skipped on cleaning this up as it is currently in `js/common` package and I didn't want to introduce a potential breaking change.

Depending on the decision from #3748 we might delete it. 

If we decided against it - I will probably want to move this off `common` and in hte `js` package as I don't think this is functionality anyone but k6 should have.